package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "stylelint-config-gds",
-  "version": "0.1.0-pre.3",
-  "description": "A work-in-progress Stylelint configuration for the UK Government Digital Service.",
+  "version": "0.1.0",
+  "description": "A Stylelint configuration for the UK Government Digital Service.",
```

Super picky - should stylelint be lowercase? Asking as that's what the project itself does when referring to itself.

```suggestion
  "description": "A stylelint configuration for the UK Government Digital Service.",
```